tpm2/constants.go

```diff
@@ -78,6 +78,18 @@ const (
 	SessionTrial  SessionType = 0x03
 )
 
+// SessionAttributes represents an attribute of a session.
+type SessionAttributes byte
+
+const (
+	AttrContinueSession SessionAttributes = 0x00000001
+	AttrAuditExclusive  SessionAttributes = 0x00000010
```

Comment:
This is hex, not binary.
So 0x00000010 becomes b0000000100000000, not b00000010

Try this https://play.golang.org/p/r_WhPYEGfIQ

Reply:
oops, thx for the example